app/assets/javascripts/google_maps.js.es6

```diff
@@ -4,6 +4,16 @@ function parseMetaTag(name) {
   return JSON.parse(content);
 }
 
+function calculateBounds(markers) {
+  const bounds = new google.maps.LatLngBounds();
+
+  markers.map(marker => {
```

Comment:
What do you think about using an `each` here since you don't care about the return value of the `map`?